app/models/user.rb

```diff
@@ -0,0 +1,34 @@
+require 'bcrypt'
+require 'dm-validations'
+
+class User
+
+  include DataMapper::Resource
+
+  attr_accessor :password_confirmation
+  attr_reader :password
+
+  has n, :peeps
+
+  property :id, Serial
+  property :email, String, unique: true
+  property :name, String
+  property :username, String
+  property :password_digest, Text
+  validates_confirmation_of :password
```

Comment:
Prefer the new style validations `validates :column, confirmation: value` over `validates_confirmation_of`.